### ISSUE TITLE
Mapbox GL JS v3 Support

### ISSUE THREAD

**Issue by jparish3** [stevage/map-gl-utils#42]
A quick test of the beta release indicates some breaking changes. Will an update to map-gl-utils be required?

**Comment by stevage** [comment]
Can you elaborate? What have you seen?

The main thing I notice is that map.getStyle() returns a very different structure with no layers. I'm still trying to work out how it behaves now, since there doesn't seem to be documentation yet,

**Comment by stevage** [comment]
Yeah, I'll keep exploring, but will probably wait for a non-beta release with documentation.

**Comment by jparish3** [comment]
will you be adding the properties from this [example](https://docs.mapbox.com/mapbox-gl-js/example/set-config-property/)?...*-emisive-strength in particular
